MC/config/analysis_testing/json/analysis-testing-mc.json

```diff
@@ -1161,5 +1165,116 @@
     "produceFBextendedTable": "true",
     "ptMax": "1e+10",
     "ptMin": "0.100000001"
+  },
+  "emcal-correction-task": {
+    "nonlinearityFunction": "MC_TestbeamFinal",
+    "logWeight": "4.5",
+    "exoticCellMinAmplitude": "4",
+    "clusterDefinition": "kV3Default",
+    "useWeightExotic": "0",
+    "disableNonLin": "0",
+    "exoticCellDiffTime": "1e+06",
+    "exoticCellInCrossMinAmplitude": "0.1",
+    "maxMatchingDistance": "0.4",
+    "exoticCellFraction": "0.97",
+    "hasShaperCorrection": "1",
+    "isMC": "0",
+    "hasPropagatedTracks": "0",
+    "processFull": "1",
```

Comment:
Is this 1 in case of MC or just processMCFull?

Reply:
As far as I understand processMCFull handles MC labels in addition to processFull, for which we don't have any consumer (at the moment). Nevertheless I switched from Full to MCFull.

Reply:
yes that is the case, we do not yet have any yet, but I think it's still safe to switch now, so we don't habe to change this again later 👍